packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -62,6 +62,13 @@ export const valueSymbol: unique symbol = Symbol("editable-tree:value");
  */
 export const anchorSymbol: unique symbol = Symbol("editable-tree:anchor");
 
+/**
+ * A symbol to get the field of a node without unwrapping in contexts where string keys are already in use for fields.
+ */
+export const getWithoutUnwrappingSymbol: unique symbol = Symbol(
```

Comment:
How about renaming it `getWrappedSymbol`?

Reply:
I thought about just `getNode` and `getField`, but for sure not `getWrapped`/`getWrappedSymbol`, since, although it is quite clear, what does "unwrapping" mean, but it's unclear (at least for me), what would "wrapped" mean. What is wrapped? Wrapped into what?

Reply:
I agree it's not very clear. I'm okay with this name, we can change it later.